OsiguSDK.Providers/Models/V1/Diagnoses.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using Newtonsoft.Json;
+
+namespace OsiguSDK.Providers.Models
+{
+    public class Diagnoses
```

Comment:
`shouldn't` this class be named `Diagnosis` instead?

Reply:
Yes, Please @dmarenco  rename that class to Diagnosis you should have to use GetListOfDiagnoses in the method because it is the plural of Diagnosis but the class itself is referring to a singular item